content/blog/monitor-application-performance-across-hybrid-cloud-environment-using-apache-skywalking-and-service-mesh.md

```diff
@@ -0,0 +1,320 @@
+
+Apache SkyWalking is lightweight and scalable. It can be easily set up as a *self-hosted* APM tool within hybrid cloud environment, without any additional external resource for hosting the tool. This can help in the context of a resource-constrained environment and remove the privacy and security concerns customers may have to put customer data going to third party services. 
+
+## Set up Apache SkyWalking
+
+### Prerequisites
+
+Before we start, make sure we meet the following requirements:
+
+* A public Kubernetes cluster from one of the public cloud providers such as *AWS*, *Microsoft Azure* or *Google*. We use one EKS cluster, named *eks-cfe-public* from AWS. However, it works if you choose a cluster from other providers.
+* A private Kubernetes cluster, named *eks-pce-clu-1* provisioned in HPE GreenLake for Private Cloud Enterprise; 
+* The *kubectl* CLI tool, version 1.23 or later, together with the *kubeconfig* files for accessing both public and private clusters;
+* The [Skupper](https://skupper.io/) CLI tool, the latest version 1.2.0. Use the [Skupper Installation](https://skupper.io/start/#step-1-install-the-skupper-command-line-tool-in-your-environment) to install this CLI tool to your local development environment;
+
+### Deploy Apache SkyWalking to AWS EKS cluster
+
+Install Apache SkyWalking using Helm charts with *elasticsearch* as storage to the public AWS EKS cluster:
+
+```markdown
+$ git clone https://github.com/apache/skywalking-kubernetes
+$ cd skywalking-kubernetes/chart
+$ helm repo add elastic https://helm.elastic.co
+$ helm dep up skywalking 
+$ kubectl create ns skywalking
+$ helm install skywalking skywalking –n skywalking \
+--set oap.image.tag=9.2.0 \
+--set ui.image.tag=9.2.0 \
+--set oap.storageType=elasticsearch \
+--set elasticsearch.imageTag=7.17.1 \
+--set elasticsearch.persistence.enabled=true \
+--set values.telemetry.v2.enabled=true \
+--set oap.envoy.als.enabled=true \
+--set oap.env.SW_ENVOY_METRIC_ALS_HTTP_ANALYSIS=mx-mesh \
+--set oap.env.SW_ENVOY_METRIC_ALS_TCP_ANALYSIS=mx-mesh
+```
+
+Above Helm commands install the Apache SkyWalking to the namespace *skywalking* of the AWS EKS cluster. It uses the *elasticsearch* as storage type and creates it as *StatefulSet* resource, running a pod on each worker node. It installs the Apache SkyWalking OAP with replicas setting as 2 to ensure the high availability. The installation enables the Envoy Access Log Service (ALS) and specifies the ALS analyzer as *mx-mesh*.
```

Comment:
It install the Apache SkyWalking Observability Analysis Platform (OAP) with replicas setting....